src/main/java/edu/cornell/kfs/pmw/batch/businessobject/PaymentWorksVendor.java

```diff
@@ -1105,7 +1110,36 @@ public void setServicesProvidedWithoutInsurance(boolean servicesProvidedWithoutI
         this.servicesProvidedWithoutInsurance = servicesProvidedWithoutInsurance;
     }
 
+    public PaymentWorksFormModeService getPaymentWorksFormModeService() {
+        if (paymentWorksFormModeService == null) {
+            paymentWorksFormModeService = SpringContext.getBean(PaymentWorksFormModeService.class);
+        }
+        return paymentWorksFormModeService;
+    }
+
+    public void setPaymentWorksFormModeService(PaymentWorksFormModeService paymentWorksFormModeService) {
+        this.paymentWorksFormModeService = paymentWorksFormModeService;
+    }
+    
+    @Override
     public String toString() {
+        if (getPaymentWorksFormModeService().shouldUseForeignFormProcessingMode()) {
+            ReflectionToStringBuilder builder = new ReflectionToStringBuilder(this, ToStringStyle.MULTI_LINE_STYLE);
```

Comment:
I like the ReflectionToStringBuilder use, this saves a lot of coding. One thing to think of is whether this is used in the reports and if it will impact what users are used to see in the reports.

Reply:
It does get printed on the report, so it will be slightly different, but with the style I selected, it should be very similar, but when we get to functional validation, I will let them know and we can get feedback.

Reply:
If it prints PaymentWorksVendor@xxxx they may not like the object address part. I am not sure what would be an easy way to remove that and keep the multi-line style.

Reply:
There is a style that supports no class name, which also prevents the Object ID ..... worse case we can create a custom style  that does both and the multi line